charts/kyverno-aws-adapter/README.md

```diff
@@ -14,20 +14,20 @@ You’ll need an [EKS](https://aws.amazon.com/eks/) cluster to run against.
 
 ### Running on the EKS cluster
 
-1. Make sure that you have configured an [IAM role for the service account](#IAM-Role-for-Service-Account) `kyverno-aws-adapter-sa` in your desired namespace (configured in `values.yaml`) and specified the role's ARN in the `roleArn` field inside `values.yaml` file.
+1. Make sure that you have configured an [IAM role for the service account](#IAM-Role-for-Service-Account) to be used by the Adapter, through the `roleArn` parameter of the Helm install command that follows.
 
 2. Add the Kyverno AWS Adapter Helm repository.
     ```console
     helm repo add kyverno-aws-adapter https://nirmata.github.io/kyverno-aws-adapter/
     helm repo update kyverno-aws-adapter
     ```
 
-3. Install the Helm chart in an appropriate namespace, providing other parameters as described in the [Values](#values) section.
+3. Install the Helm chart in an appropriate namespace, providing other parameters as described in the [Values](#values) section. Parameters `roleArn`, `eksCluster.name`, `eksCluster.region`  are required.
```

Comment:
As suggested in the TODO of this PR #35 , maybe we can move to a separate `local-dev.md` file. And it would be actually nice to point to `values.yaml`, because its local development.

You can have instructions for someone to build and run the controllers manually too.

Reply:
Yes, will raise an issue to have a dev guide. Does the above change look ok for the helm install though? Because, there is no chart.yaml, user could use -set or -f.

Reply:
yes. lgtm